TreeMaker/python/makeTreeFromMiniAOD_cff.py

```diff
@@ -157,15 +157,15 @@ def makeTreeFromMiniAOD(self,process):
                 genCollection = cms.InputTag("prunedGenParticles"),
                 debug = cms.bool(False),
                 # Particles we want to save from the decay chain of the tops
-                childIds = cms.vint32(1,2,3,4,5,11,12,13,14,15,16,24),
+                childIds = cms.vint32(1,2,3,4,5,6,11,12,13,14,15,16,24),
                 # Particles we want to save the last copy from the hard scatter
                 parentIds = cms.vint32(
                     6,22,23,24,25,
                     1000021,1000022,1000023,1000024,1000025,1000035,1000037,1000039,
                     1000001,1000002,1000003,1000004,1000005,1000006,
                     2000001,2000002,2000003,2000004,2000005,2000006,
-                    4900021,4900023,4900101,4900102,4900103,4900111,4900113,4900211,4900213,51,52,53,
-                    5000001,5000002,
+                    4900001,4900021,4900023,4900101,4900102,4900103,4900111,4900113,4900211,4900213,51,52,53,
```

Comment:
Does it make sense to add all the scalar mediators here? (also 4900002, etc.)

Reply:
I guess it depends on how future proof we want the TreeMaker code to be? Right now, we are setting 4900001 to be the only mediator that can be probed bu LHC, do we expect anyone to use the other scale mediators?

Reply:
I think the MadGraph t-channel setup can produce the other mediators.

Reply:
OK, I guess this change doesn't taking up more space in the output anyway, for the sake of completion we might as well add them now.